src/odemis/acq/align/light.py

```diff
@@ -80,6 +80,7 @@ def _doTurnOnLight(f, bl, ccd):
         # Turn the light on, full power!
         bl.power.value = bl.power.range[1]
         while True:
+            time.sleep(0.1)
```

Comment:
Have you verified this is enough to resolve the test on Ubuntu 20/Python 3.8?

Reply:
I could reproduce the issue (memory leak) on my computer, and this solved it. Anyway, for now it's just a good "side-effect". I am working separately on actually fixing the memory leak (and making a proper test case that checks for that). So I don't care too much on whether that will fix really the test case (yet) on a specific Ubuntu.